### PR TITLE
Chapter1/Homework 10

### PR DESCRIPTION
**Pull request by Tailor-AI**
First draft of homework 10.

### CODE REVIEW

File: Src/BootCamp.Chapter/CaesarCipher.cs [hunk 1]
```diff
@@ -11,12 +10,46 @@ public static class CaesarCipher
...
+            return new string(asciiChars);
         }
 
         public static string Decrypt(string message, byte shift)
         {
-            return message;
+            return string.IsNullOrEmpty(message) ? message : DecryptedMessage(message, shift);
+        }
+
+        private static string DecryptedMessage(string message, byte shift)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/CaesarCipher.cs]: Decrypt is the same as encrypt (except the shift is different). Please refactor

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter/CaesarCipher.cs (hunk 1 above)
Comment (Tailor-AI) [reply Src/BootCamp.Chapter/CaesarCipher.cs]: Thanks for the feedback. I added a bool parameter to EncryptedMessage to check if it should encrypt or decrypt.

File: Src/BootCamp.Chapter/CaesarCipher.cs [hunk 2]
```diff
@@ -10,39 +9,29 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return string.IsNullOrEmpty(message) ? message : EncryptedMessage(message,shift); 
+            return string.IsNullOrEmpty(message) ? message : EncryptedMessage(message, shift, true); 
         }
 
-        private static string EncryptedMessage(string message, byte shift)
+        private static string EncryptedMessage(string message, byte shift, bool shiftRight)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/CaesarCipher.cs]: No no 😄 
Have both decrypt and encrypt, no need an if statement.
But decrypt should reuse encrypt, that's all.

Comment (Tailor-AI) [reply Src/BootCamp.Chapter/CaesarCipher.cs]: Ok :)
Hopefully now I got it right?

**Review by Almantask — APPROVED** [review]
Yup, well done now 😄

File: Src/BootCamp.Chapter/CaesarCipher.cs (hunk 2 above)
Comment (Almantask) [reply Src/BootCamp.Chapter/CaesarCipher.cs]: Yes

**Pull request closed without merging**